src/Jackett.Common/Indexers/XSpeeds.cs

```diff
@@ -216,19 +215,24 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
                 IsConfigured = false;
                 throw;
             }
+
             return IndexerConfigurationStatus.RequiresTesting;
         }
 
         protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query)
         {
-            var releases = new List<ReleaseInfo>();
             var searchString = query.GetQueryString();
             var prevCook = CookieHeader + "";
 
-            var searchParams = new Dictionary<string, string> {
+            var categoryMapping = MapTorznabCapsToTrackers(query);
+
+            var searchParams = new Dictionary<string, string>
+            {
                 { "do", "search" },
-                { "category", "0" },
-                { "include_dead_torrents", "no" }
+                { "category", categoryMapping.FirstIfSingleOrDefault("0") }, // multi category search not supported
+                { "include_dead_torrents", "no" },
+                { "sort", "added" },
+                { "order", "desc" }
```

Comment:
Are some or all of `title`, `size`, and `seeders` available as options, as well as `ascending`? If so, can you make this user-configurable? e.g.:
https://github.com/Jackett/Jackett/blob/cf91a35999c52f906eebbc0996fb96d0d556233d/src/Jackett.Common/Indexers/Anidex.cs#L75-L92